### PR TITLE
[new] New MFA on RO docs, edits to MFA index

### PR DESCRIPTION
**Pull request by Catografix**

### CODE REVIEW

**Comment by Catografix** [comment]
@ntotten Do I have to do something special when adding a new doc?
404
https://auth0-docs-content-pr-1466.herokuapp.com/docs/multifactor-authentication/ro

**Comment by Catografix** [comment]
I thought this was going to conflict with https://github.com/auth0/docs/pull/1484
But that PR makes changes to /multifactor-authentication/index2.md

Why are there 2 similar index files?

**Comment by Catografix** [comment]
These changes are ok for now. Angela is reworking this doc for new authenticators. #1484

File: articles/multifactor-authentication/index.md [hunk 1]
```diff
@@ -26,15 +33,13 @@ Auth0 provides built-in support for the following one-time password authenticati
...
-
-> Auth0 also supports plugging in other multifactor providers. If you already have your own provider, click here to use a different provider.
+You may enable these options in the [Multifactor Auth](${uiURL}/#/multifactor) section of the Auth0 Dashboard.
 
 #### MFA Using Google Authenticator
 
 Google Authenticator allows you to request six- to eight-digit one-time use password as the second factor after your user has attempted to log in with their Google credentials.
 
-You will find further instructions on enabling this feature [here](/multifactor-authentication/google-authenticator).
+For instructions on enabling this featur, see:  [Configuring MFA Using Google Authenticator](/multifactor-authentication/google-authenticator).
```
Comment (mcastany) [inline articles/multifactor-authentication/index.md]: typo

File: articles/multifactor-authentication/ro.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+description: How to enable MFA on the Resource Owner endpoint.
+---
+
+# MFA on a Resource Owner endpoint
+
+To enable MFA on the Resource Owner /ro endpoint, include the `options.mfa.active` flag in the connection and set it to `true`. 
+
+To enable enrollment, also include the `options.mfa.return_enroll_settings` flag in the connection.
+
+### SCENARIO 1 - Enrolled user with Google authentication and a valid code
```
Comment (ntotten) [inline articles/multifactor-authentication/ro.md]: Why all the capitalization? I haven't seen this before I dont think.

Comment (Catografix) [inline articles/multifactor-authentication/ro.md]: Oh, not intentional. Will fix.

File: articles/multifactor-authentication/ro.md [hunk 3]
```diff
@@ -0,0 +1,225 @@
+---
+title: MFA on a Resource Owner endpoint
+description: How to enable MFA on the Resource Owner endpoint.
+---
+
+# MFA on a Resource Owner endpoint
+
+To enable MFA on the Resource Owner [/ro](/api/authentication#!#post--oauth-ro) endpoint, include the `options.mfa.active` flag in the connection and set it to `true`.
```
Comment (Catografix) [inline articles/multifactor-authentication/ro.md:8]: @mcastany Where is this flag? Its not in the endpoint call, is it a setting in the dashboard?

**Comment by Catografix** [comment]
@ntotten The RO doc is not being served.
https://auth0-docs-content-pr-1466.herokuapp.com/docs/multifactor-authentication/ro
I also get an error in my local deployment, but I dont know what is wrong:

ReferenceError: provider is not defined
    at Object.eval [as template](lodash.templateSources[187]:9:10)
    at renderContent (/Users/Catowork/Documents/auth0-docs/lib/docs/renderer.js:61:18)
    at processSections (/Users/Catowork/Documents/auth0-docs/lib/docs/renderer.js:70:17)
    at renderDoc (/Users/Catowork/Documents/auth0-docs/lib/docs/index.js:53:27)
    at Layer.handle [as handle_request](/Users/Catowork/Documents/auth0-docs/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/Catowork/Documents/auth0-docs/node_modules/express/lib/router/route.js:131:13)
    at Route.dispatch (/Users/Catowork/Documents/auth0-docs/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request](/Users/Catowork/Documents/auth0-docs/node_modules/express/lib/router/layer.js:95:5)
    at /Users/Catowork/Documents/auth0-docs/node_modules/express/lib/router/index.js:277:22
    at Function.process_params (/Users/Catowork/Documents/auth0-docs/node_modules/express/lib/router/index.js:330:12)

**Comment by Catografix** [comment]
I will incorporate these changes into the new docs, and fix whatever is wrong with the ro doc that is keeping it from being served.

**Comment by ntotten** [comment]
The issue is with the doc there is something wrong with the rendering.

**Comment by Catografix** [comment]
I thought so, am going to rebuilt it from scratch, maybe a stray character somewhere.

**Pull request closed without merging**